Makefile

```diff
@@ -92,6 +92,9 @@ clean:
 	rm -f $(SPEC)
 	rm -f po/LINGUAS
 
+clean-all:
+	git clean -f
```

Comment:
to properly clean everything, you also need to remove directories (`-d`), and ignored files (`-x`) as the build artifacts are in `.gitignore`; so it would be `git clean -fdx`

Reply:
Updated

Reply:
I still find it redundant of what is doable already with standard git comands; see my earlier note.